scheduler/Dockerfile.dataflow

```diff
@@ -14,7 +14,7 @@ RUN gradle build --no-daemon --info
 ################################################################################
 
 # Some dependencies require glibc, which Alpine does not provide
-FROM registry.access.redhat.com/ubi9/openjdk-17-runtime:1.16-3
+FROM registry.access.redhat.com/ubi9/openjdk-17-runtime:1.17-1.1705573249
```

Comment:
This change is related to the dataflow libraries, might be worth to remove it from this PR as I'll have it my PR.

In my changes, I've updated this to `registry.access.redhat.com/ubi9/openjdk-17-runtime:1.17` following the idea I've mentioned above. In fact, I can see dependabot pinning us to even more specific versions here than what we had before.

Reply:
ok moved to `openjdk-17-runtime:1.17` in [348301f](https://github.com/SeldonIO/seldon-core/pull/5234/commits/348301f94878a160711144c4d624c8504e582ac8)